packages/block-library/src/post-comment-avatar/block.json

```diff
@@ -0,0 +1,25 @@
+{
+	"apiVersion": 2,
+	"name": "core/post-comment-avatar",
+	"title": "Post Comment Avatar",
+	"category": "design",
+	"parent": [ "core/post-comment" ],
+	"description": "Post comment avatar.",
+	"textdomain": "default",
+	"usesContext": [ "commentId" ],
+	"supports": {
+		"html": true,
```

Comment:
Should this be disabled since it's mostly a dynamic block?

Reply:
I'm using commentId to get Avatar info. Is this required? Should I remove those lines?

Reply:
Sorry, I mean the `html: true` which enables the html editor for the block.

Reply:
Done!